src/Features/LanguageServer/ProtocolUnitTests/Workspaces/LspWorkspaceManagerTests.cs

```diff
@@ -413,8 +454,8 @@ public async Task TestWorkspaceEventUpdatesCorrectWorkspaceWithMultipleWorkspace
         Assert.Equal(firstDocument, lspDocument);
     }
 
-    [Fact]
-    public async Task TestSeparateWorkspaceManagerPerServerAsync()
+    [Theory, CombinatorialData]
+    public async Task TestSeparateWorkspaceManagerPerServerAsync(bool mutatingLspWorkspace)
```

Comment:
It this scenario in the mutating world even a sensible scenario?

Reply:
it's possibly not.  def one of hte odder cases.  but i don't mind documenting what actually happens :)